GeneralsMD/Code/GameEngineDevice/Source/W3DDevice/GameClient/W3DView.cpp

```diff
@@ -1924,10 +1924,26 @@ void W3DView::setAngleAndPitchToDefault( void )
 //-------------------------------------------------------------------------------------------------
 void W3DView::setDefaultView(Real pitch, Real angle, Real maxHeight)
 {
+	// TheSuperHackers @fix Mauller adjust the max camera height to compensate for the screen aspect ratio
+	Real baseAspectRatio = (Real)DEFAULT_DISPLAY_WIDTH / (Real)DEFAULT_DISPLAY_HEIGHT;
+	Real currentAspectRatio = TheDisplay->getAspectRatio();
```

Comment:
I would not mind doing `TheDisplay->getWidth() / TheDisplay->getHeight()` here and skipping on the new `getAspectRatio` virtual function.